water-alternatives.csl

```diff
@@ -424,27 +437,21 @@
             </group>
             <text macro="publisher"/>
           </group>
-          <choose>
-            <if variable="event-date">
-              <date variable="event-date">
-                <date-part name="day" suffix=" " range-delimiter="-"/>
-                <date-part name="month" suffix=" "/>
-                <date-part name="year" range-delimiter="/"/>
-              </date>
-            </if>
-            <else>
-              <date variable="issued">
-                <date-part name="day" suffix=" " range-delimiter="-"/>
-                <date-part name="month" suffix=" "/>
-                <date-part name="year" range-delimiter="/"/>
-              </date>
-            </else>
-          </choose>
+          <date variable="event-date">
+            <date-part name="day" suffix=" " range-delimiter="-"/>
+            <date-part name="month" suffix=" "/>
+            <date-part name="year" range-delimiter="/"/>
+          </date>
         </group>
       </else-if>
       <else-if type="interview personal_communication" match="any">
         <group delimiter=". " prefix=". ">
-          <text variable="medium" text-case="capitalize-first"/>
+          <choose>
+            <if variable="title" match="none">
+              <text value="Personal communication"/>
+            </if>
+          </choose>
+          <text variable="medium" text-case="lowercase" prefix="By "/>
```

Comment:
No, it is not odd. I think medium item contain "e-mail", "instant message" etc. And then the output from the template is correct to the guide for authors: Harvey, J.H.M. 1999. Personal communication. By email. 12 October 1999.